src/frontend/src/components/common/CheckBox.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+
+const Checkbox = ({
+  label,
+  checked,
+  value,
+  onChange,
+  ...props
+}: {
+  label: string;
+  checked: any;
```

Comment:
can we please avoid `any` here? Seems like something specific that we are passing.

Reply:
I would like to avoid `any` at all costs. It renders typescript pointless.

Reply:
@binggaofw - will you be taking care of this in your next PR?